Doc/library/dbm.rst

```diff
@@ -58,41 +58,33 @@ the Oracle Berkeley DB.
 
 .. function:: open(file, flag='r', mode=0o666)
 
-   Open the database file *file* and return a corresponding object.
+   Open a database and return the corresponding database object.
 
-   If the database file already exists, the :func:`whichdb` function is used to
-   determine its type and the appropriate module is used; if it does not exist,
-   the first module listed above that can be imported is used.
+   :param file:
+      The database file to open.
 
-   The optional *flag* argument can be:
-
-   .. csv-table::
-      :header: "Value", "Meaning"
+      If the database file already exists, the :func:`whichdb` function is used to
+      determine its type and the appropriate module is used; if it does not exist,
+      the first submodule listed above that can be imported is used.
+   :type file: :term:`path-like object`
 
-      ``'r'`` (default), |flag_r|
-      ``'w'``, |flag_w|
-      ``'c'``, |flag_c|
-      ``'n'``, |flag_n|
+   :param str flag:
+      * ``'r'`` (default), |flag_r|
+      * ``'w'``, |flag_w|
+      * ``'c'``, |flag_c|
+      * ``'n'``, |flag_n|
```

Comment:
A list of flags renders nicer and more compact than a table, when embedded in the param list.